orchestrator/test_runner/src/transaction_stress_test.rs

```diff
@@ -107,22 +106,27 @@ pub async fn transaction_stress_test(
                 if !found {
                     good = false;
                 }
+                if good {
+                    break;
```

Comment:
I think this will cause the `for` loop to exit early in a lot of cases. I believe the original `break` was to exit the outer `while Instant::now()` loop.

Reply:
This commit: https://github.com/PeggyJV/gravity-bridge/pull/274/commits/4d7cc8f741f554794da2abf6e8eb4feb9bf9cf78 fixes this, resolving this comment now. Thank you.